_blog.yml

```
@@ -1453,3 +1453,13 @@
   tags:
     - guide
     - expert-acceleration-program
+
+- local: vq-diffusion
+  title: "VQ Diffusion with 🧨 Diffusers"
+  author: williamberman
+  thumbnail: /blog/assets/117_vq_diffusion/thumbnail.png
+  date: November 23, 2022
```

```suggestion
  date: November 28, 2022
```

Let's merge on Monday?

---

I'm not sure if we got an affirmative from Shuyang and I think we should check with @pcuenca that his feedback was all adequately addressed. Let's double check with them before merging, but besides that I'm good with whenever!